main.ts

```diff
@@ -149,13 +154,15 @@ export class ImporterModal extends Modal {
 					el.addEventListener('click', async () => {
 						let progress = new ProgressReporter();
 						this.modalEl.addClass('is-loading');
+						new Notice('Import started.');
```

Comment:
Let's not add stuff to the main script

Reply:
Fixed and pushed.
FYI. This was added because if the user is uploading a large number of files, it will appear to them that the program has hung - since there is no message and interface stays the same.

There is now https://github.com/obsidianmd/obsidian-importer/issues/35, but until that's implemented the issue will persist. Hence I implemented it in main as a placeholder because it shouldn't be implemented individually by importers.